db/src/commonMain/kotlin/com/islamversity/db/model/Calligraphy.kt

```diff
@@ -0,0 +1,89 @@
+package com.islamversity.db.model
+
+import com.squareup.sqldelight.ColumnAdapter
+
+inline class LanguageCode(val lang: String) {
+
+    val isValid: Boolean
```

Comment:
Can be `private`.

Reply:
I need other people to be able to verify